src/Api.js

```diff
@@ -564,6 +564,18 @@ export default class Api extends ToolApi {
     return toolDataPathExistsSync(dataPath);
   }
 
+  getisVerseEdited(chapter, verse) {
```

Comment:
Should this method name follow camel case convention?

Reply:
Nice catch